atom/browser/autofill/atom_autofill_client.cc

```diff
@@ -196,7 +196,7 @@ void AtomAutofillClient::UpdateAutofillPopupDataListValues(
 }
 
 void AtomAutofillClient::HideAutofillPopup() {
-  if (api_web_contents_) {
+  if (api_web_contents_ && api_web_contents_->IsFocused()) {
```

Comment:
I'm not sure if this is the best way to handle this. I think the root of the problem is that we don't check the tabId before clearing the contextMenuDetail state in browser-laptop `WINDOW_AUTOFILL_POPUP_HIDDEN`, but if we are going to keep something like this I think it should track the current popup_visible state by setting it along with the calls to `OnPopupShown` and `OnPopupHidden`